### PR TITLE
Add merchants with only one item

### PR DESCRIPTION
**Pull request by jbranham1**
**What did you change?**
Added merchants_with_only_one_item and merchants_with_only_one_item_registered_in_month
**Next Steps**
Finish iteration 4
**Known Issues**

### CODE REVIEW

File: lib/merchant.rb [hunk 1]
```diff
@@ -20,4 +22,8 @@ def items
   def invoices
     @invoices ||= @repository.invoices_by_merchant_id(@id)
   end
+
+  def one_item?
+    items.count == 1
+  end
```
Comment (abreaux26) [inline lib/merchant.rb:28]: @jbranham1 how are you getting items here?

**Review by abreaux26 — COMMENTED** [review]
Everything looks great! Nice work! 
I just have one question

File: lib/merchant.rb (hunk 1 above)
Comment (jbranham1) [reply lib/merchant.rb:28]: @abreaux26 we have a method called items on the merchant!

Comment (abreaux26) [reply lib/merchant.rb:28]: duhh... it was late last night and forgot to view the whole file 😅

**Review by abreaux26 — APPROVED** [review]

**Pull request merged**